thrift/src/main/thrift/contentatom.thrift

```diff
@@ -56,6 +58,7 @@ union AtomData {
   14: commonsdivision.CommonsDivision commonsDivision
   15: chart.ChartAtom chart
   16: audio.AudioAtom audio
+  17: emailsignup.EmailSignUpAtom emailsignup
```

Comment:
should we camelCase this, similar to `commonsDivision` above?

Reply:
There is a mixture of camel case (commonsDivision) and non-camel case (storyquestions) here, so leaving as is